src/charts/donut.js

```diff
@@ -282,18 +287,60 @@ define(function(require){
             svg.select('.donut-text').text('');
         }
 
+        /**
+         * Find the slice by id and growth it if needed
+         * @private
+         */
+        function initHighlightSlice() {
+            if (!highlightedSliceId && highlightedSliceId !== 0) return;
```

Comment:
I was thinking that maybe we could compare `highlightedSliceId` with undefined, so that indexes from 0 to N would be valid.

Nit: Could you add curly braces? We usually don't do one-lines.
Thanks!